$OpenBSD: patch-taxipilot_movingobject_cpp,v 1.3 2017/08/31 19:58:21 espie Exp $
Can't take the address of an unnamed temporary, clang insists.

Index: taxipilot/movingobject.cpp
--- taxipilot/movingobject.cpp.orig
+++ taxipilot/movingobject.cpp
@@ -242,8 +242,11 @@ void MovingObject::read_object(const QDomElement *desc
 					new_action->jump = true;
 					new_action->jump_target = 	cdp->get_int_attribute ("target", e_b, 1, max_segment, 1, 1) -1;	// start counting from 1 instead of 0
 				}
-				new_action->conditional = new Conditional (&cdp->get_element ("condition", e_b, 0), level);
+				{
+				QDomElement tmp = cdp->get_element("condition", e_b, 0);
+				new_action->conditional = new Conditional (&tmp, level);
 				new_action->next = 0;
+				}
 
 				if (!first_action) {		// find place to insert the action
 					Action *action_dummy = path[segments].action;
